metis-indexing/src/main/java/eu/europeana/indexing/fullbean/AggregationFieldInput.java

```diff
@@ -20,22 +20,36 @@
  */
 final class AggregationFieldInput implements Function<Aggregation, AggregationImpl> {
 
-  private final Supplier<List<WebResourceImpl>> webResourcesSupplier;
+  private final List<WebResourceImpl> recordWebResources;
+  private final List<WebResourceImpl> referencedWebResources;
```

Comment:
The code below would read a lot better if these were maps (with the about values as keys). Do you think that would be a useful change?

Reply:
In fact, on further inspection, we never need the actual referenced web resources again, so `referencedWebResources` could even be a set of about values.

Reply:
Okay so I don't see much of a benefit for maps but I changed the first one to a map to do some extra work, why not :D
We need to keep in mind now though that duplicate web resources with the same about would be discarded and only the first encountered will be kept. That was happening anyway automatically from mongo but now it will be "cleared" before hand.(In general that shouldn't be allowed at the first place anyway)
For the second I opted out with a set of strings instead.